examples/one_dot_one/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "devDependencies": {
     "@types/node": "^18.0.0",
-    "typescript": "^5.2.2"
+    "typescript": "^4.9.0"
```

Comment:
We should probably use MRL to make sure we don't break the invariant of certain packages being 4x